tcms/testruns/views.py

```diff
@@ -280,9 +281,6 @@ class EditTestRunView(UpdateView):
 
     def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
-        context['test_plan'] = self.object.plan
-        context['builds'] = Build.objects.filter(product=self.object.plan.product, is_active=True)
-
         return context
```

Comment:
form_class on line 280 should be NewRunForm. Not sure why the template works with the old form though. Maybe UpdateView filters all ModelChoiceFields before hand.

NB: when you change to a NewRunForm it will complain that the `case` field is not available.
NB2: also needs adjustment for .populate()